mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -44,23 +61,31 @@
   private static final String NAME = "name";
   private static final String LOCATION = "location";
 
+  private static final Set<String> PROPERTIES_TO_REMOVE = Stream
+      .of(InputFormatConfig.TABLE_SCHEMA, InputFormatConfig.PARTITION_SPEC, LOCATION, NAME)
+      .collect(Collectors.toCollection(HashSet::new));
+
   private Catalogs() {
   }
 
   /**
    * Load an Iceberg table using the catalog and table identifier (or table path) specified by the configuration.
-   * Catalog resolution happens in this order:
-   * 1. Custom catalog if specified by {@link InputFormatConfig#CATALOG_LOADER_CLASS}
-   * 2. Hadoop or Hive catalog if specified by {@link InputFormatConfig#CATALOG}
-   * 3. Hadoop Tables
    * @param conf a Hadoop conf
    * @return an Iceberg table
    */
   public static Table loadTable(Configuration conf) {
     return loadTable(conf, conf.get(InputFormatConfig.TABLE_IDENTIFIER), conf.get(InputFormatConfig.TABLE_LOCATION));
   }
 
-  // For use in HiveIcebergSerDe and HiveIcebergStorageHandler
+  /**
+   * Load an Iceberg table using the catalog specified by the configuration.
+   * The table identifier ({@link Catalogs#NAME}) or table path ({@link Catalogs#LOCATION}) should be specified by
+   * the controlling properties.
+   * Used by HiveIcebergSerDe and HiveIcebergStorageHandler
```

Comment:
Minor: Javadoc won't automatically break paragraphs, so you have to use `<p>` between them.

Reply:
Happily relearning javadoc formatting 😄 
Done